pb-cli.js

```diff
@@ -418,7 +449,24 @@ var talkResp = function (error, response, body) {
     }
 }
 
-var chatResp = function(error, response, body) {
+var writeLogFile = (input, output) => {
+    var arr = [];
+    var note = {
+      input,
+      output
+    };
+    try {
+      var resLogString = fs.readFileSync('Response-log.json');
+      arr = JSON.parse(resLogString);
+    } catch(e) {
+    }
+    arr.push(note);
+    fs.writeFileSync('Response-log.json', JSON.stringify(arr));
+};
+
+var updatedchatResp = function(input){
```

Comment:
Another little nitpick - `updatedChatResp` would be the conventional way of writing this function name.